docker-compose-cdc-mssql-polling.yml

```diff
@@ -20,6 +20,6 @@ services:
       SPRING_PROFILES_ACTIVE: ${SPRING_PROFILES_ACTIVE}
 
       EVENTUATE_CDC_CLEANER_1_PIPELINE: default
-      EVENTUATE_CDC_CLEANER_1_PURGE_MESSAGESENABLED: "true"
-      EVENTUATE_CDC_CLEANER_1_PURGE_MESSAGESMAXAGEINSECONDS: 1
-      EVENTUATE_CDC_CLEANER_1_PURGE_INTERVALINSECONDS: 1
\ No newline at end of file
+      EVENTUATE_CDC_CLEANER_1_CLEAN_MESSAGESENABLED: "true"
```

Comment:
Could the naming be improved: `EVENTUATE_CDC_CLEANER_1_CLEAN_INTERVALINSECONDS `:

* Is `_CLEAN_` needed?
* Can `INTERVALINSECONDS` be `INTERVAL_IN_SECONDS`?

Reply:
1) Yes, it is property group https://github.com/eventuate-foundation/eventuate-cdc/blob/b761138b0d48246d5b6c9a27e50f0e03b681a4f3/eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/common/properties/MessageCleanerProperties.java#L18
2) Probably I need to change property mapping for it, I will check

Reply:
> 1. Yes, it is property group

Why? Couldn't the properties of `MessageCleaningProperties` be moved/copied into `MessageCleanerProperties`?

Reply:
ok, will do